examples/hello_world_component/CMakeLists.txt

```diff
@@ -33,5 +33,5 @@ if(EXISTS "${HPX_DIR}")
     COMPONENT_DEPENDENCIES iostreams
   )
 else()
-  message(WARNING "HPX_DIR=${HPX_DIR} does not exist. Did you forget to run the install rule?")
+  message(FATAL_ERROR "HPX_DIR=${HPX_DIR} does not exist. Did you forget to run the install rule?")
```

Comment:
This should not be a fatal error. I still want to run the unit tests without installing

Reply:
Fair point, but then we have to be very careful to make sure that we don't end up back to where it's run before installation and we don't notice errors.

Reply:
I've changed this back to a warning (same in the pkgconfig test).

Reply:
Well, it boils down to how we setup our testers. The rostam testers did install before running the unit tests, for example.